.travis.yml

```diff
@@ -49,7 +49,7 @@ env:
     - JHI_LIB_BRANCH=master
     # if JHI_GEN_BRANCH value is release, use the release from NPM
     - JHI_GEN_REPO=https://github.com/jhipster/generator-jhipster.git
-    - JHI_GEN_BRANCH=master
+    - JHI_GEN_BRANCH=spring-boot_2.1.0
```

Comment:
as spring-boot_2.1.0 branch in generator-jhipster project has been merged, we can switch here to master

Reply:
switched back to master generator branch